src/components/RequestBox/RequestBox.js

```diff
@@ -635,12 +662,94 @@ export default class RequestBox extends Component {
     );
   }
 
+  /**
+   * Relaunch DTR using the available context
+   */
+  relaunch = (e) => {
+    this.buildLaunchLink()
+      .then(link => {
+        //e.preventDefault();
+        window.open(link.url, "_blank");
+      });
+  }
+
+  buildLaunchLink() {
+    // build appContext and URL encode it
+    let appContext = "";
+    let order = undefined, coverage = undefined, response = undefined;
+
+    if (!this.isOrderNotSelected()) {
+      if (Object.keys(this.state.deviceRequest).length > 0) {
+        order = `${this.props.ehrUrl}/${this.state.deviceRequest.resourceType}/${this.state.deviceRequest.id}`;
```

Comment:
all of these references can be local.  Won't make much of a difference, but this is how it's written in the IG.